src/stack.py

```diff
@@ -0,0 +1,24 @@
+"""Implementation of Stack data type."""
+
+from linked_list import LinkedList
+
+
+class Stack(object):
+    """Class representation of a stack."""
+
+    def __init__(self, iterable=None):
+        """Instantiate stack."""
+        self.linked_list = LinkedList(iterable)
+        self.length = self.linked_list.length
+        self.head_node = self.linked_list.head_node
```

Comment:
If we want to check the value of the head node, we need to assign it in the Stack, correct?

I suppose we could get it using my_stack.linked_list.headnode, but isnt that kind of ugly?